users.js

```diff
@@ -674,6 +674,9 @@ class User {
 			this.connections[i].send(initdata);
 		}
 		this.named = false;
+		for (let i in this.games) {
+			this.games[i].renamePlayer(this, oldid);
```

Comment:
I think `onRename` might be a better name for this function... is it already used elsewhere or something?